views/update/review-beneficial-owner-individual.html

```diff
@@ -65,6 +65,7 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">Review the individual beneficial o
       {% set isBeneficialOwnerGov = false %}
       {% set natureOfControlText = "What is their nature of control?" %}
       {% set beneficial_owner_nature_of_control_types = beneficial_owner_nature_of_control_types %}
+      {% set trustees_nature_of_control_types = trustees_nature_of_control_types %}
```

Comment:
Can I ask what this is doing? From what I can see, nothing? 🙈 Same with the other view?

Reply:
It was removed as part of 678, but I suppose if we can't do a digital update while the OE has trusts then...yeah this is useless :D